### DISCUSSION TITLE
Why use inline style on popover trigger?

### DISCUSSION THREAD

**Discussion by cassiebrooks** [Akryum/v-tooltip#363] [Q&A]
Is there a reason for having `display: inline-block` inlined on the Popover `.trigger` div? It means I have to use an `!important` to override it.

https://github.com/Akryum/v-tooltip/blob/fa672f6d6d2004d9cb285ce4551f12c9ca1f6540/src/components/Popover.vue#L9

If there's no reason for it, would moving it to the CSS be considered a breaking change?

Thanks for the lib!

**Comment by Livijn** [comment]
I have the same question/issue.

**Comment by yvesh** [comment]
Agree, that should be an option maybe, but not hardcoded. :-) Thank you!
